### ISSUE TITLE
Can't compile in Linux

### ISSUE THREAD

**Issue by ABelliqueux** [Lameguy64/smxtool#1]
Hi, I'm trying to compile under Linux but can't figure out how to . I tried with Netbeans and "make" in the cli, but I end up with :

`$ make -f nbproject/Makefile-Release-linux.mk dist/Release-linux/GNU-Linux/smxtool`

`make: *** No rule to make target '../common/common.cpp', needed by 'build/Release-linux/GNU-Linux/_ext/b4460d1a/common.o'.  Stop.`

Any hint ?

EDIT:  Fixed it. The includes paths were incorrect and "colorpreview.cpp" should be capitalized : ColorPreview.cpp .

See this commit for the corrections : https://github.com/ABelliqueux/smxtool/commit/fa5ce235eb36043bed36b443313cb6fef4cd05af

**Comment by ABelliqueux** [comment]
I submitted another PR (https://github.com/Lameguy64/smxtool/pull/3) to further fix compiling on Linux.

Here is a branch with those fixes : https://github.com/ABelliqueux/smxtool/tree/linux-build 

I think I can close the issue once the PR is merged ?
